UI/OptionsWnd.cpp

```diff
@@ -133,7 +133,7 @@ namespace {
             fs::path path = FilenameToPath(file);
             return boost::algorithm::ends_with(file, STRINGTABLE_FILE_SUFFIX) &&
                 fs::exists(path) && !fs::is_directory(path);
-        } catch (std::exception ex) {
+        } catch (const std::exception&) {
```

Comment:
none of the catch blocks in this file actually use the exception, so I don't think the change matters... but if updating, could switch to just `catch (...)`

Reply:
If I understand correctly, the purpose of this patch is to silence compiler warnings, which should be achieved, regardless of the catch blocks using the exceptions or not.

So, good to merge?